presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/FileBasedAccessControl.java

```diff
@@ -455,7 +457,16 @@ public void checkCanExecuteProcedure(ConnectorSecurityContext context, SchemaRou
     @Override
     public Optional<ViewExpression> getColumnMask(ConnectorSecurityContext context, SchemaTableName tableName, String columnName, Type type)
     {
-        return Optional.empty();
+        if (INFORMATION_SCHEMA_NAME.equals(tableName.getSchemaName())) {
```

Comment:
That requires documentation that you cannot set masks for `information_schema`. Maybe we should be fail on initialization then if user specifies any rule against `information_schema`?